core/src/zeit/simplecast/tests/test_connection.py

```diff
@@ -89,6 +89,38 @@ def test_simplecast_gets_podcast_folder(self):
         container = self.simplecast.folder(self.episode_info['created_at'])
         self.assertEqual(container, self.repository['podcasts']['2023-08'])
 
+    def test_update_published_or_unpublished_episodes_should_publish_or_retract_audio_object(self):
+        boolsets = [
+            (True, True, True, False),
+            (True, False, False, True),
+            (False, True, True, False),
+            (False, False, False, False),
+        ]
+        for bs in boolsets:
```

Comment:
Vorschlag: für eine bessere Lesbarkeit schlage ich folgende Variablennamen vor :)

```suggestion
        for initial_state, updated_state, publish_expected, retract_expected in boolsets:
```

Reply:
Ich wusste, dass da was kommt 😂